Lib/glyphsLib/__init__.py

```diff
@@ -132,14 +132,17 @@ def build_masters(
         if create_background_layers:
             ufo_create_background_layer_for_all_glyphs(source.font)
 
-        ufo_path = os.path.join(master_dir, source.filename)
-        clean_ufo(ufo_path)
-        source.font.save(ufo_path)
+        # Save UFO to disk in a clean way, unless we are looking at an intermediate
+        # layer, which we will hit on some other iteration.
+        if not source.layerName:
```

Comment:
elsewhere you first check if `hasattr(source, "layerName")`; let's make it consistent (cf https://github.com/googlei18n/glyphsLib/pull/478/files#r245102549)